Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -323,147 +413,269 @@ function Enable-StorageEncryptionInTransit
         catch
         {
             Write-Host "Please load Helper.ps1 file in current PowerShell session before executing the script." -ForegroundColor $([Constants]::MessageType.Error)
-            break
+            Write-Host $([Constants]::SingleDashLine)
+            return
         }
 
-        if($resourceResolver.messageToPrint -ne $null)
+        if($null -ne $resourceResolver.messageToPrint)
         {
             $resourceSummary += "Excluded resource/resource group summary:`n " 
             $resourceSummary += $resourceResolver.messageToPrint
         }  
-        Write-Host $resourceSummary
+        Write-Host $resourceSummary  
+        Write-Host $([Constants]::SingleDashLine)
     }
-    
-    Write-Host "Total Storage Account(s) excluded  from remediation:" [$($totalStorageAccount - ($storageAccounts | Measure-Object).Count)] -ForegroundColor $([Constants]::MessageType.Update)
+
+    Write-Host "Total Storage Account(s) excluded from remediation:" [$($totalStorageAccount - ($storageAccounts | Measure-Object).Count)] -ForegroundColor $([Constants]::MessageType.Update)
```

Comment:
In Auto remediation excluded resource count will always be zero, so shall we move this line within if block itself?

Reply:
done